humanguide/design-concepts/in-app-icons-design.rst

```diff
@@ -0,0 +1,72 @@
+In-app icons design
+===================
+
+Interface Icons
+---------------
+
+Interface icons communicate either status or action displays, such as battery level or an alarm, whilst also enabling the user to perform actions like opening a menu, sharing content or print a document. The interface icons are located in the top bar or in-app icons, such as in the Header or the menus.
```

Comment:
The first sentence in this paragraph runs on. Consider rewriting it to two sentences with distinct points. Also, "printing a document".